x-pack/plugins/log_explorer/public/components/flyout_detail/types.ts

```diff
@@ -4,13 +4,17 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-
-import type { DataView } from '@kbn/data-views-plugin/common';
-import type { FlyoutContentProps } from '@kbn/discover-plugin/public';
 import type { DataTableRecord } from '@kbn/discover-utils/types';
+import type { DocViewRenderProps } from '@kbn/unified-doc-viewer/types';
 
-export interface FlyoutProps extends FlyoutContentProps {
```

Comment:
Nit: It looks like we're not really using `FlyoutContentActions` from `src/plugins/discover/public/customizations/customization_types/flyout_customization.ts` any more... Could we consolidate these types definitions?

Reply:
I derived with a new commit the same actions set for the customization from the `DocViewRenderProps`, let me know if it looks better after removing the `FlyoutContentActions` type